Apps/Admin/Tests/Functional/.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
     "env": {
-        "cypress/globals": true
+        "cypress/globals": true,
+        "es2021": true
```

Comment:
Why is there an Admin change for functional tests?  And what is this change for?